patches/server/0066-Cadmium.patch

```diff
@@ -0,0 +1,1177 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Simon Gardling <titaniumtown@gmail.com>
```

Comment:
Shouldn't it be the email of LucilleTea (and you as co-author ?)

Reply:
Fixed